### DISCUSSION TITLE
Support for multi-line tabs and changing the color of the folder and its contents.

### DISCUSSION THREAD

**Discussion by 7jrxt42BxFZo4iAnN4CX** [zed-industries/zed#15369] [Feature Requests]
multi-line tabs:

 ![94047686-0ff64e80-fdf0-11ea-863d-220a41c28010](https://github.com/user-attachments/assets/d016f4e2-7ca8-4445-9bd7-fbc0a18843b0)

Colored folders/tabs:
<img width="1019" alt="243261401-2aa9ba17-c25e-40b0-b478-4da80d6a3b93" src="https://github.com/user-attachments/assets/dc104730-59da-44c9-8604-46080db3ad0c">
<img width="1057" alt="243370814-45f713a0-f1a1-42d2-bf44-206400ceaa3c" src="https://github.com/user-attachments/assets/f60bd808-baaf-42aa-a842-8a0196c43ade">

**Comment by Red-Devil-SDO** [comment]
Also voting for this. Because for me, I very sorrow, that Zed miss this functionality. I often open a lot of tabs, and switch between them, just have one row - it's very uncomfortable and hard. It would be great, if in Zed implements multi-row functionality for opened tabs with files. Or if it very complicated, just add some list with opened files. It would be great if you add this functionality. Thank you.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
